crypto/curve25519/curve25519.c

```diff
@@ -4618,7 +4624,10 @@ static void GFp_fe_mul121666(fe h, fe f) {
   h[9] = (int32_t)h9;
 }
 
-static void x25519_scalar_mask(uint8_t e[32]) {
+/* Prevent -Wmissing-prototypes warnings. */
+void GFp_x25519_scalar_mask(uint8_t e[32]);
```

Comment:
This function is only compiled if `BORINGSSL_X25519_X86_64` is not defined, which is why I kept it here. Would you like the prototype to be unconditionally declared?

I think this is _untrue_ for `GFp_ed25519_scalar_mask()`, though, so I'll fix that either way.